.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
   - curl ifconfig.co|xargs echo "Travis IP address is ";
 
 script:
-  - "mvn cobertura:cobertura"
+  - "mvn fmt:check verify cobertura:cobertura"
```

Comment:
Don't think we need the verify here because there are no integration tests for this repo. It will increase the build time by running the tests twice otherwise